lua/pac3/core/client/parts/command.lua

```diff
@@ -25,7 +25,10 @@ end
 
 function PART:OnShow(from_rendering)
 	if not from_rendering and self:GetExecuteOnShow() then
-		self:Execute()
+		timer.Simple(0, function()
+			if self.Hide or self:IsHidden() then return end
```

Comment:
am i correct thinking that this does not execute the command when hidden from rendering, but does execute when hidden with event?

Reply:
I don't think so. But idk, I haven't thought that far. There was an issue where it wrongly auto-triggered with timerx events or something. In the gmod wiki, I read a few times that a 0 timer can be used for various things. Therefore I tried that and I think it worked.